### ISSUE TITLE
Freeze issues

### ISSUE THREAD

**Issue by Brumazur** [GrowthcraftCE/Growthcraft-1.12#367]
Hello, I manage a modded minecraft server and since we installed growthcraft many of our players are experiencing loss of fps and freezes. We tried to determine the cause but impossible, we are running on a high performance vps and the problem cannot come from them. Ideas ? Playing on the latest version of forge using also spongeforge and your latest release

**Comment by Alatyami** [comment]
Which version of growthcraft are you running? Whats the filename of the jar?

**Comment by Brumazur** [comment]
growthcraft-1.12.2-4.2.2.1.jar !

**Comment by Alatyami** [comment]
Thanks. Do you have a link to your modpack? We're not seeing this in other packs so we'll need to do some testing and profiling.

**Comment by Brumazur** [comment]
Yes of course, here it is and thank you very much for your time. https://www.dropbox.com/s/6k46ioigvab78g8/Modpack.zip?dl=1
